package.json

```diff
@@ -13,11 +13,18 @@
   "author": "wtw",
   "license": "ISC",
   "dependencies": {
-    "@actions/core": "^1.10.0"
+    "@actions/core": "^1.10.0",
+    "@actions/github": "^6.0.0",
+    "@octokit/action": "^6.0.7",
+    "@octokit/graphql": "^7.0.2",
+    "@octokit/rest": "^20.0.2",
+    "axios": "^0.24.0",
```

Comment:
Is axios actually needed?

Reply:
This is a holdover from the template. I'll remove it.